### PR TITLE
Added factorial using recursion

### PR DESCRIPTION
**Pull request by Pratik180198**
Changes are made. Please add the label "Hactoberfest"

### CODE REVIEW

File: factorial_02.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
```
Comment (pythonites) [inline factorial_02.py:2]: Please explain what is 'try' in multiline comment at the end or start of program

File: factorial_02.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    def factorial(num): #Using Recursive Method
+        if num == 0 or num ==1:
+            return 1
+        else:
+            return num * factorial(num-1)
+    print(f"Factorial of {num} is {factorial(num)}") #Recursive Method Answer
+except:
```
Comment (pythonites) [inline factorial_02.py:9]: Please explain about except in multiline comments at the end of program

File: factorial_02.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
+    def factorial(num): #Using Recursive Method
+        if num == 0 or num ==1:
+            return 1
+        else:
+            return num * factorial(num-1)
```
Comment (pythonites) [inline factorial_02.py:7]: Please add comment about return keyword at the end of program

**Review by pythonites — CHANGES_REQUESTED** [review]
Please make necessary changes as requested.
Marking as 'hacktober-accepted'

File: factorial_02.py (hunk 1 above)
Comment (Pratik180198) [inline factorial_02.py:2]: #The try block lets you test a block of code for errors.

Comment (Pratik180198) [reply factorial_02.py:2]: #The try block lets you test a block of code for errors.

File: factorial_02.py (hunk 3 above)
Comment (Pratik180198) [reply factorial_02.py:7]: #The return keyword is to exit a function and return a value.

File: factorial_02.py (hunk 2 above)
Comment (Pratik180198) [reply factorial_02.py:9]: #The except block lets you handle the error.

File: factorial_02.py (hunk 1 above)
Comment (Pratik180198) [reply factorial_02.py:2]: #The try block lets you test a block of code for errors.

**Review by Pratik180198 — COMMENTED** [review]
comments added for try, except and return.

**Pull request merged**